Classes/Shared/Views/Common/Lottie/LottieImageView.swift

```diff
@@ -51,8 +51,10 @@ extension LottieImageView {
         animationView.animation = LottieAnimation.named(jsonName)
     }
 
-    func play(with configuration: LottieImageView.Configuration) {
-        animationView.play(fromProgress: configuration.from, toProgress: configuration.to, loopMode: configuration.loopMode)
+    func play(with configuration: LottieImageView.Configuration, completion: ((Bool) -> Void)? = nil) {
```

Comment:
Suggestion. Try to avoid using async closures. Try to use reactive (for data streams) or async/await approach (for fire-once actions).